components/Competition/GameList/StyledGameList.js

```diff
@@ -29,7 +29,7 @@ export default function StyledGameListItemComponent({
             : `/competition/${competition.id}/game-detail/${game.gameId}`
         }
       >
-        Edit game ✏️
+        Edit game <EditIcon disabled={false} />
```

Comment:
If this `EditIcon` is always set to be visible by `disabled={false}`, why adding this prop at all?

Reply:
I am using the icon on a link and a button component where I am changing the color based on if the user can click the corresponding button. I just had a look and can "save" the disabled prop when changing the ternary operator the other way around.
Thanks for the good hint! 😊